File: 백준/Bronze/14910. 오르막/오르막.java
import java.io.*;

public class Main {
    public static void main(String[] args) throws IOException {
        BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
        BufferedWriter bw = new BufferedWriter(new OutputStreamWriter(System.out));
        String result = "Good";
        String[] input = br.readLine().trim().split(" ");
        int temp = -1000001;

        for (int i = 0; i < input.length; i++) {
            int num = Integer.parseInt(input[i]);
            if (num >= temp) {
                temp = num;
            } else {
                result = "Bad";
                break;
            }
        }

        bw.write(result);
        bw.flush();
        br.close();
        bw.close();
    }
}

File: 백준/Bronze/14910. 오르막/오르막.py
import sys

input = sys.stdin.readline

nums = list(map(int, input().strip().split()))
temp = -1000001
result = "Good"

for num in nums:
    if num >= temp:
        temp = num
    else:
        result = "Bad"
        break

sys.stdout.write(f"{result}" + '\n')